cpp/benchmarks/io/orc/orc_reader_multithreaded.cpp

```diff
@@ -112,13 +112,11 @@ void BM_orc_multithreaded_read_common(nvbench::state& state,
                    cudf::io::read_orc(read_opts, stream, rmm::mr::get_current_device_resource());
                  };
 
-                 threads.paused = true;
-                 for (size_t i = 0; i < num_files; ++i) {
-                   threads.submit(read_func, i);
-                 }
+                 threads.pause();
```

Comment:
`pause` and `unpause` methods are only enabled when this macro `BS_THREAD_POOL_ENABLE_PAUSE` is explicitly defined. Reference link [here ](https://github.com/bshoshany/thread-pool/blob/097aa718f25d44315cadb80b407144ad455ee4f9/include/BS_thread_pool.hpp#L419).